README.md

```diff
@@ -42,6 +42,25 @@ location. If that is not available, the environment variable
 `LIBZMQ_INCLUDE_DIR`) can be defined to avoid the invocation of
 `pkg-config`.
 
+Note for Windows users (re. dynamic linking of ZeroMQ):
+
+- When building `libzmq` from sources, the library must be renamed 
+  to `zmq.lib` from the auto named `libzmq-v***-mt-gd-*_*_*.lib`, 
+  `libzmq.lib`, `libzmq-mt-*_*_*.lib`, etc. 
```

Comment:
Stylistic note: I assume you intend to have a paragraph break after `etc.`. If so, you need a separate blank line. If not, you should your IDE/editor's facilities to block-justify the whole (then single) paragraph. Right now, it looks like a missing blank line (i.e., paragraph break). Similiar instances below. </nitpick>.

Reply:
Markdown will render this as close spaced bullets. With empty lines between the bullets the spacing increases. See GitHub screenshot below:

![image](https://user-images.githubusercontent.com/39146854/58254893-142dda00-7d6c-11e9-9852-2a8caacb3e2a.png)

Reply:
Oh, sorry, I missed this dashes next to the plus signs of the diff.